backend/dal/Repositories/Interfaces/ILeaseService.cs

```diff
@@ -11,7 +11,7 @@ public interface ILeaseService : IRepository<PimsLease>
     {
         int Count();
         IEnumerable<PimsLease> Get(LeaseFilter filter);
-        PimsLease GetFlat(long id, bool asNoTracking = false);
+        long GetRowVersion(long id);
```

Comment:
I see that we now have two folders: `dal/Repositories` and `dal/Services` - wouldn't it make sense to name the repo classes interface something like `WhateverRepo` instead of `WhateverService`. Seems a tad confusing

Reply:
agreed, we should rename all of the files in dal/repositories to whateverrepo.